payjoin/src/send/mod.rs

```diff
@@ -362,8 +362,10 @@ impl V2PostContext {
         self,
         response: &mut impl std::io::Read,
```

Comment:
In `V2GetContext` this was changed to a `&[u8]`. Is there a reason to not change it here too?